example-runner/pom.xml

```diff
@@ -49,11 +49,15 @@
     </scm>
 
     <properties>
+        <jackson.version>2.11.0</jackson.version>
```

Comment:
Looks like this was not meant to be changed

Reply:
Just moved to the top in line with the style used in other poms, dependency versions first, then dockerfile things last